library/src/scala/quoted/Type.scala

```diff
@@ -9,6 +9,8 @@ abstract class Type[T] extends Quoted {
 /** Some basic type tags, currently incomplete */
 object Type {
 
+  implicit def apply[T]: Type[T] = ???
```

Comment:
Are calls to this methods going to be replaced by the tags below when possible and replaced `'[T]` if the type is statically known?